### ISSUE TITLE
google_datafusion_instance add support for accelerators

### ISSUE THREAD

**Issue by Guimove** [hashicorp/terraform-provider-google#9070]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment. If the issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If the issue is assigned to a user, that user is claiming responsibility for the issue. If the issue is assigned to "hashibot", a community member has claimed the issue already.

### Description

During the creation of a Data Fusion Instance, we can enable some "accelerators" (actually only replication). This is not supported by the provider for the moment.

### New or Affected Resource(s)

* google_datafusion_instance 

### Potential Terraform Configuration

```tf
resource "google_data_fusion_instance" "instance" {
  provider                      = google-beta
  name                          = "datafusion"
  description                   = "datafusion instance"
  region                        = "europe-west-2"
  type                          = "developer"
  enable_stackdriver_logging    = false
  enable_stackdriver_monitoring = false
  private_instance              = false
  version                       = "6.4.0"
  accelerators {
    "replication",
    "list_of_accelerators"
  }
}
```

### References

* https://cloud.google.com/data-fusion/docs/concepts/replication
* https://cloud.google.com/data-fusion/docs/reference/rest/v1beta1/projects.locations.instances

![image](https://user-images.githubusercontent.com/1172067/117028289-dc709080-acfd-11eb-893b-49d9ca0d1dc0.png)

**Comment by Guimove** [comment]
Hello, 

Do you have any news about that feature ? 

Actually, I’m using a custom null_resource to run after the instance creation : 

```
curl -H "Authorization: Bearer $(gcloud auth print-access-token)" \
-H "Content-Type: application/json" \
${CDAP_ENDPOINT}  -X PATCH -d '{ "accelerators":[{"accelerator_type":"CDC", "state":"ENABLED"}]}'
```

But this is not really flexible.

This is major for us

**Comment by shuyama1** [comment]
Waiting for b/199551875 to be fixed.

**Comment by dmccaffery** [comment]
@shuyama1 - I am happy to take this on if b/199551875 has been resolved. I cannot find any reference to an issue with that id (I assume its an internal bug). If work on this is ready to go, I am happy to submit a PR -- just let me know.

**Comment by a1031** [comment]
@dmccaffery Indeed, it seems like `b/199551875` is an internal/non-public bug on the [Google Issue Tracker for Cloud Data Fusion](https://issuetracker.google.com/issues?q=status:open%20componentid:716116&s=created_time:desc) - if I visit https://issuetracker.google.com/issues/199551875 I get `Access denied for: <email>`, and if I visit the same link but with a higher number that wouldn't exist I get `Issue <number> does not exist` (example: https://issuetracker.google.com/issues/999551875).

**Comment by melinath** [comment]
b/245981973
